Monika After Story/game/script-topics.rpy

```diff
@@ -12999,3 +12999,31 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_intrusivethought",category=['Psychology'],prompt="Intrusive thoughts",random=True))
+
+label monika_intrusivethought:
+    m 1euc "Hey, [player]..."
+    m 1ekd "Do you ever get thoughts that are upsetting, distressing, or all around unpleasant to you?"
+    m 3eub "I've been reading on a study on what's called 'intrusive thought'. It's actually quite interesting." 
+    m 3ekc "The study involves that the mind tends to think of some... unpleasant thoughts when triggered by certain, often negative circumstances."
```

Comment:
The beginning of this line sounds a bit odd in my opinion, I've never seen "involve" used like that in a sentence. Perhaps replace "involves" by "claims", or something similar?

@Rai99 Thoughts on this?